assets/wizards/site-design/views/main/index.js

```diff
@@ -69,6 +75,27 @@ const Main = ( {
 				description={ __( 'Activate a theme you like to get started', 'newspack' ) }
 			/>
 			<ThemeSelection theme={ themeSlug } updateTheme={ updateThemeSlug } />
+			{ isPartOfSetup && homepagePatterns.length > 0 ? (
+				<>
+					<SectionHeader
+						title={ __( 'Homepage', 'newspack' ) }
+						description={ __( 'Choose your favorite homepage layout', 'newspack' ) }
```

Comment:
```suggestion
						description={ __( 'Choose a homepage layout', 'newspack' ) }
```

Nit: it's not really about what my favorite is, since I'm probably part of a larger organization.

Reply:
Good suggestion, added in b4ff0ca